pkg/lintcontext/context.go

```diff
@@ -1,22 +1,66 @@
 package lintcontext
 
 import (
+	"encoding/json"
+
 	"golang.stackrox.io/kube-linter/internal/k8sutil"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
 // ObjectMetadata is metadata about an object.
 type ObjectMetadata struct {
 	FilePath string
-	Raw      []byte
+	Raw      []byte `json:"-"`
 }
 
 // An Object references an object that is loaded from a YAML file.
 type Object struct {
 	Metadata  ObjectMetadata
-	K8sObject k8sutil.Object
+	K8sObject k8sutil.Object `json:"-"`
+}
+
+// K8sObjectName contains just enough information about k8s object to allow identifying and locating it.
+type K8sObjectName struct {
+	Namespace, Name, GroupVersionKind string
+}
+
+// GetK8sObjectName allows extracting object names from Object.K8sObject.
+func (o *Object) GetK8sObjectName() K8sObjectName {
+	return K8sObjectName{
+		Namespace:        o.K8sObject.GetNamespace(),
+		Name:             o.K8sObject.GetName(),
+		GroupVersionKind: o.K8sObject.GetObjectKind().GroupVersionKind().String(),
+	}
 }
 
+// String provides plain-text representation of K8sObjectName.
+func (n K8sObjectName) String() string {
+	ns := n.Namespace
+	if ns == "" {
+		ns = "<no namespace>"
+	}
```

Comment:
```suggestion
	ns := stringutils.OrDefault(n.Namespace, "<no namespace>")
```

Reply:
This is so neat. I regret that I found out about it only now.